crates/bevy_render/src/camera/projection.rs

```diff
@@ -63,12 +63,17 @@ pub enum ScalingMode {
     /// Manually specify left/right/top/bottom values.
     /// Ignore window resizing; the image will stretch.
     None,
-    /// Match the window size. 1 world unit = 1 pixel.
-    WindowSize,
+    /// Match the window size.
+    /// The argument is the amount of world units per pixel.
```

Comment:
I don't like the design of this API. It feels hard to use, and like something that should be handled by cameras.